app/src/components/searchWidget/searchWidgetWrapper.tsx

```diff
@@ -96,7 +96,7 @@ class SearchWidgetWrapper extends React.Component<StateChangedProps, any> {
                     <Autocomplete
                         id="pickup-location"
                         className="pickup-location"
-                        placeholder="Sydney International Airport (SYD)"
+                        defaultValue="Sydney International Airport, Australia"
                         minValueLength={3}
```

Comment:
No magic strings please
Move this into .env file and refer it from here. 
Plus add logic if empty to put an empty string

see how .env value is being used 

REACT_APP_TRINGO_API

(your .env vlue should start with REACT_APP_ prefix)